boards/xenvm-qemu.overlay

```diff
@@ -0,0 +1,47 @@
+
+#include <mem.h>
+
+/ {
+	chosen {
+		zephyr,flash = &flash_sim0;
+	};
+
+	flashcontroller0: flashcontroller {
+		compatible = "zephyr,sim-flash";
+		label = "FLASH_SIMULATOR";
+		#address-cells = <1>;
+		#size-cells = <1>;
+		erase-value = <0xff>;
+		flash_sim0: flash_sim@0 {
+			compatible = "soc-nv-flash";
+			reg = <0x00000000 DT_SIZE_M(1)>;
+			erase-block-size = <1024>;
+			write-block-size = <4>;
+			partitions {
+				compatible = "fixed-partitions";
+				#address-cells = <1>;
+				#size-cells = <1>;
+			/*
+			 * Storage partition will be used by FCB/LittleFS/NVS
+			 * if enabled.
+			 */
+				storage_partition: partition@1000 {
+					label = "storage";
```

Comment:
some notes from Zephyr documentation:
```
Earlier versions of Zephyr made frequent use of the label property, which is distinct from the 
tandard node label. Use of the label property in new devicetree bindings, as well as use of the
DT_LABEL macro in new code, are actively discouraged. Label properties continue to persist
for historical reasons in some existing bindings and overlays, but should not be used in new
bindings or device implementations.
```

moreover, I have seen warnings form `west` that `label` property is a deprecated during compilation of a project where I had DTS labels.

Reply:
This is partition on flash simulator and added here for test purpose on qemu. On product branch we will not use it.

Reply:
could you please add todo that we have to delete this chunk of code before merging it to product branch?

Reply:
I afraid that we can forget about deleting this code in the future
but ok, I got your point, thanks